src/Tribe/Attendee_Repository.php

```diff
@@ -88,6 +106,21 @@ public function __construct() {
 			'rsvp_status'           => [ $this, 'filter_by_rsvp_status' ],
 		] );
 
+		// Add object default aliases.
+		$this->update_fields_aliases = array_merge( $this->update_fields_aliases, [
+			'ticket_id'      => '_tribe_tickets_ticket_id',
```

Comment:
Maybe adding a getter function that returns the default args? Would using `wp_parse_args` an option?

Reply:
Intention is to only support an array for `$this->update_fields_aliases `, so the usefulness of `wp_parse_args` is limited greatly here -- I went for a plain `array_merge` as a result of that

Reply:
As for the getter, I'm just following the way all of the repositories are laid out right now, the getter could be useful to set up a pattern for in the near future though